client/signup/utils.js

```diff
@@ -93,6 +95,25 @@ function getNextStepName( flowName, currentStepName ) {
 	return flow.steps[ indexOf( flow.steps, currentStepName ) + 1 ];
 }
 
+
+function autoFillField( config ) {
+	let { form , stepName, fieldToTake, fieldToMerge, signupProgressStore } = config;
+	if ( ! signupProgressStore ) {
+		return form;
+	}
+	let siteStepProgress = find(
```

Comment:
We try to use `const` instead of `let` for all identifiers we only assign to once (which, I believe, is all of the ones in this function).